### ISSUE TITLE
Switching language does not take effect

### ISSUE THREAD

**Issue by Lxb921006** [tendenci/tendenci#1013]
When I switch to Japanese, only part of it works

**Comment by jennyq** [comment]
Yes, indeed. This is because Japanese hasn't been fully translated yet and we're lack of Japanese translators. If you know the language and would like to help, you are welcome to join us on transifex https://www.transifex.com/tendenci/.
